plugins/ddf-plugins/src/main/kotlin/org/codice/compliance/saml/plugin/ddf/IdpResponderProvider.kt

```diff
@@ -126,8 +128,9 @@ class IdpResponderProvider : IdpResponder {
                 .replace("window.idpState = ", "")
                 .replace(";", "")
 
-        val queryParams = ObjectMapper()
-                .readValue(idpState, MutableMap::class.java) as MutableMap<String, String>
+        val gson = Gson()
+        GsonBuilder().setPrettyPrinting().create()
```

Comment:
❓ What does this line do? Seems unnecessary.

Reply:
It just adds the"pretty print" feature. I can remove it cause we don't really use it right now